lib/hound/helpers/log.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Hound.Helpers.Log do
+  @moduledoc "Functions to work with the log"
+
+  import Hound.RequestUtils
+
+  @doc """
+  Fetches console.log() from the browser as a single string of log-lines.
+  """
+  def fetch_log() do
+    fail_if_webdriver_selenium("fetch_log()")
```

Comment:
As I think we will be using this in some other places,
I just added a helper to handle more easily.

Could you please rebase on top of master and replace this line with

```elixir
Hound.NotSupportedError.raise_for(%{driver: "selenium", browser: "firefox"})
```

also, you will need to add

```elixir
require Hound.NotSupportedError
```

somewhere at the top of the module, as `raise_for` is a macro.